### ISSUE TITLE
 torch.onnx.dynamo_export fails with torch.hamming_window

### ISSUE THREAD

**Issue by D4ve-R** [pytorch/pytorch#120316]
### 🐛 Describe the bug

Using `torch.onnx.dynamo_export` with `torch.hamming_window` in `pytorch=2.1.1` fails.
```python
import torch

class Test(torch.nn.Module):
    def __init__(self):
        super(Test, self).__init__()
    def forward(self, x: torch.Tensor):
        window = torch.hamming_window(400)
        return x * window

torch.onnx.dynamo_export(Test(), torch.randn(1,400))
```
Fails with 
```bash
RuntimeErrorWithDiagnostic: Unsupported FX nodes: {'call_function': ['aten.hamming_window.default']}. 
```
According to the docs  
https://github.com/pytorch/pytorch/blob/main/docs/source/jit_unsupported.rst#id33  
`requires_grad` is not supported, so I'm confused, is `torch.hamming_window` supported?

Full sarif log attached 

[report_dynamo_export.txt](https://github.com/pytorch/pytorch/files/14361362/report_dynamo_export.txt)

### Versions

PyTorch version: 2.1.1
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: macOS 14.1.2 (x86_64)
GCC version: Could not collect
Clang version: 15.0.0 (clang-1500.0.40.1)
CMake version: version 3.25.2
Libc version: N/A

Python version: 3.11.1 (v3.11.1:a7a450f84a, Dec  6 2022, 15:24:06) [Clang 13.0.0 (clang-1300.0.29.30)] (64-bit runtime)
Python platform: macOS-14.1.2-x86_64-i386-64bit
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz

Versions of relevant libraries:
[pip3] numpy==1.26.2
[pip3] onnx==1.15.0
[pip3] onnxruntime==1.17.0
[pip3] onnxscript==0.1.0.dev20240221
[pip3] pytorch-lightning==2.1.4
[pip3] pytorch-metric-learning==2.4.1
[pip3] torch==2.1.1
[pip3] torch-audiomentations==0.11.0
[pip3] torch-pitch-shift==1.2.4
[pip3] torchaudio==2.1.1
[pip3] torchmetrics==1.3.0.post0
[pip3] torchvision==0.16.1
[conda] Could not collect
